external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -87,6 +87,16 @@
     <PackageToInclude Include="$(XUnitRunnerPackageId)" />
   </ItemGroup>
 
+  <!-- System.Data.SqlClient tests require sni.dll in the test folder -->
+  <ItemGroup Condition="'$(TargetGroup)' == 'uapaot'">
```

Comment:
If you see my other comment on that other PR, you'll see how because of binplacing to ILC folder, doing it in the runtime.depproj would require additional settup as we won't want to binplace everything on to that folder.